Xamarin.Forms.Core/Image.cs

```diff
@@ -21,6 +21,17 @@ public class Image : View, IImageController, IElementConfiguration<Image>
 
 		public static readonly BindableProperty IsLoadingProperty = IsLoadingPropertyKey.BindableProperty;
 
+		public enum AnimationPlayBehaviorValue
```

Comment:
how about  `bool IsAutoPlay` instead? Is the `None`  value meanigful?
Even so, naming should be improved. It's a bit ambigous. And `Value` suffix can be ommited.

Reply:
I agree with @rookiejava, that enum is probably misnamed, and might not be required at all

Reply:
Agree, enum came from original issue, have switched to an bool auto play property, as suggested.